lib/datadog/open_feature/noop_evaluator.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require_relative 'ext'
+require_relative 'resolution_details'
+
+module Datadog
+  module OpenFeature
+    # This class is a noop interface of evaluation logic
+    class NoopEvaluator
```

Comment:
As a placeholder, I think it's fine; but I'm curious, are we going to remove the `NoopEvaluator` once the actual one gets put in? Or not? May be worth documenting the expected future of this class in a comment here.